src/groups/mwc/mwcu/mwcu_operationchain_cpp03.cpp

```diff
@@ -1,23 +1,8 @@
-// Copyright 2022-2023 Bloomberg Finance L.P.
-// SPDX-License-Identifier: Apache-2.0
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
```

Comment:
Need to keep this license header

Reply:
this file is automatically generated, the BMQ team should be the one to figure this stuff out
see related issue: {DRQS 175682820}